fabric-rendering-v1/src/client/java/net/fabricmc/fabric/api/client/rendering/v1/IdentifiedLayer.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.rendering.v1;
+
+import net.minecraft.client.gui.LayeredDrawer;
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.fabric.impl.client.rendering.WrappedLayer;
+
+/**
+ * A hud layer that has an identifier attached for use in {@link LayeredDrawerWrapper}.
+ *
+ * <p>For common use cases, see {@link LayeredDrawerWrapper}.
+ */
+public interface IdentifiedLayer extends LayeredDrawer.Layer {
```

Comment:
The documentation of this class should state that its static final fields are in the order in which their corresponding layer is rendered.

Additionally, I think the documentation should make clear what the default render predicate for each layer is, that this API does not allow changing the existing predicate for a layer, and that operations relative to a layer will generally inherit that layer's predicate. The last two points might belong in the documentation of `LayeredDrawerWrapper` instead.

Reply:
Done in 3384eae.